build.sbt

```diff
@@ -137,23 +217,35 @@ lazy val testsInput = project
     skip in publish := true
   )
 
-val isScala213 = Def.setting {
-  VersionNumber(scalaVersion.value).matchesSemVer(SemanticSelector(">=2.13"))
-}
-
 def scala212LibraryDependencies(deps: List[ModuleID]) = List(
   libraryDependencies ++= {
-    if (isScala213.value) Nil
+    if (isScala213.value || isScala3.value) Nil
     else deps
   }
 )
+val tests = project
```

Comment:
Added a root test project for utility classes - `BaseSuite.scala` and `Compat.scala`